FluentTerminal.App.ViewModels/OverlayViewModel.cs

```diff
@@ -0,0 +1,54 @@
+﻿using FluentTerminal.App.Services;
+using GalaSoft.MvvmLight;
+using System;
+
+namespace FluentTerminal.App.ViewModels
+{
+    public class OverlayViewModel : ViewModelBase
+    {
+        private readonly IDispatcherTimer _overlayTimer;
+        private bool _showOverlay;
+        private string _overlayContent;
+
+        public OverlayViewModel(IDispatcherTimer dispatcherTimer)
+        {
+            _overlayTimer = dispatcherTimer;
+            _overlayTimer.Interval = new TimeSpan(0, 0, 2);
+            _overlayTimer.Tick += OnResizeOverlayTimerFinished;
+        }
+
+        public bool ShowOverlay
+        {
+            get => _showOverlay;
+            set
+            {
+                Set(ref _showOverlay, value);
+                if (value)
+                {
+                    if (_overlayTimer.IsEnabled)
```

Comment:
It's a bit too much logic for a setter. I think the usage would be easier to understand if there was a method `Show(string message)` which sets our values and starts the timer. We might also make the setters private to avoid confusion

Reply:
I was thinking since every time i set the value, i want to dispatch the timer, i thought it would be more efficient to keep it in the set. However, i see what you mean and thus fixed this in the newest commit. I now call Show() instead.